src/Cli/dotnet/ParseResultExtensions.cs

```diff
@@ -127,14 +127,25 @@ public static bool UsingRunCommandShorthandProjectOption(this ParseResult parseR
 
         public static IEnumerable<string> GetRunCommandShorthandProjectValues(this ParseResult parseResult)
         {
-            var properties = parseResult.ValueForOption(RunCommandParser.PropertyOption);
+            var properties = GetRunPropertyOptions(parseResult, true);
             return properties.Where(property => !property.Contains("="));
         }
 
         public static IEnumerable<string> GetRunCommandShorthandPropertyValues(this ParseResult parseResult)
```

Comment:
`GetRunCommandShorthandPropertyValues` doesn't seem like the best name here, because it returns both the properties that were passed using the shorthand and longhand form.